.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,105 @@ body:
+        > by this constraint due to the way semantic versioning works for `0.x` packages. This is intentional, see
+        > https://semver.org/ for details.
+      label: Updating packages
+      options:
+        - label: I have updated all `@opentelemetry/*` packages but the issue persists
+          required: true
+  - type: dropdown
+    id: esm
+    attributes:
+      description: |
+        **How to find this information:**
+          - check your `package.json`
+            - if you have `"type": "commonjs"` or no `"type"` then you're likely using CommonJS
+            - if you have `"type": "module"`, you are likely using ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.
+          - other indicators:
+            - using `.js` or `.cjs` (CommonJS)
+            - using `.mjs` (ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.)
+      label: Which module type are you using?
+      multiple: false
+      options:
+        - CommonJS
+        - ESM
+        - I don't know
+  - type: checkboxes
+    id: bundling
+    attributes:
+      description: |
+        Bundling removes `require` and `import` statements, which results in most `@opentelemetry/instrumentation-*` packages not working as they rely on the presence of these statements to function.
+      label: Bundling and `@opentelemetry/instrumentation-*`packages
+      options:
+        - label: I understand that `@opentelemetry/instrumentation-*` packages are not intended to work in bundled apps (with a few exceptions).
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Reproduction
   - type: textarea
     attributes:
       label: What happened?
-      description: Please provide as much detail as you reasonably can.
+      description: |
+        > [!IMPORTANT]
+        > **Please provide as much detail as you reasonably can.**
+        > We all know the complexities of developing with JavaScript/TypeScript. **Be specific** about your setup and **DO NOT** assume that the person handling your bug report knows how to use your specific combination of tooling. Always provide build instructions and config files.
       value: |
         ## Steps to Reproduce
-
+          <!--
+            Please consider providing a link to a reproducer repository. It is extremely difficult to exactly reproduce problems
+            and doing so can take multiple hours for us. By providing a reproducer you indirectly contribute back to the project
+            and ensure we can get working on fixing your bug as soon as possible instead of spending precious time asking
+            questions and checking responses. You know your tooling and setup and we know OpenTelemetry JavaScript internals.
```

Comment:
```suggestion
            The most effective way to solve a bug is to provide a link to a reproducer repository. It is extremely difficult to exactly reproduce problems
            and doing so can take multiple hours for us. You know your tooling and setup and we know OpenTelemetry JavaScript internals.
```

Reply:
yes it does 🧐🎩 
your thanks for the suggestion, accepted 🙂